src/sentry/issues/escalating.py

```diff
@@ -333,6 +335,15 @@ def manage_issue_states(
             )
             if data and activity_data:
                 data.update(activity_data)
+            if data and snooze_details:
+                try:
+                    jsonschema.validate(snooze_details, INBOX_REASON_DETAILS)
+
+                except jsonschema.ValidationError:
+                    logging.error(f"Expired snooze_details invalid jsonschema: {snooze_details}")
```

Comment:
I'd recommend including `snooze_details` in `extra` rather than as part of the message so that these group properly